src/components/biggive-accordion-entry/biggive-accordion-entry.tsx

```diff
@@ -0,0 +1,16 @@
+import { Component, Prop, Element } from '@stencil/core';
+
+@Component({
+  tag: 'biggive-accordion-entry',
+  styleUrl: 'biggive-accordion-entry.scss',
+  shadow: true,
+})
+export class BiggiveAccordionEntry {
+  @Element() host: HTMLBiggiveAccordionEntryElement;
+
+  @Prop() entryHeading: string = '';
```

Comment:
`entryHeading` doesn't seem to be used anywhere. Should it be used inside `render`?

Reply:
This is read by the biggive-accordion parent. There is no rendered output from biggive-accordion-entry. The content is outputted by the parent so that we have more control over the CSS from the parent.

Reply:
Ah yes I see its read here https://github.com/thebiggive/components/blob/8a1f852ef06be0fc50ee9a655374ad90348d00d2/src/components/biggive-accordion/biggive-accordion.tsx#L45 

For some reason my PhpStorm doesn't find that usage if I search for it from here, it only finds it if I search for it from `components.d.ts`.

Thanks.